docs/ops.html

```diff
@@ -3615,7 +3615,7 @@ <h4 class="anchor-heading"><a id="kraft_zk_migration" class="anchor-link"></a><a
   <p>The following features are not yet supported for ZK to KRaft migrations:</p>
 
   <ul>
-    <li>Downgrading to ZooKeeper mode during or after the migration</li>
+    <li>Reverting to ZooKeeper mode after the migration</li>
     <li>Other features <a href="#kraft_missing">not yet supported in KRaft</a></li>
   </ul>
```

Comment:
I'm not sure we should say "Reverting to ZooKeeper mode after the migration" since the implication is that this isn't supported **yet**.  In fact it will never be supported.  So maybe this should just point to the list of unsupported features only.

Reply:
Good point. I reworked this section a bit.

Reply:
I'm wondering if my above comment still holds.  If we say "The following features are not yet supported for ZK to KRaft migrations: Reverting to ZooKeeper mode after the migration" then it seems to be implied that we will support it at some point (because we said "are not yet supported"). There will never be support for reverting to ZooKeeper mode after the migration is finalized without data loss.